pype/plugins/nuke/publish/collect_writes.py

```diff
@@ -143,5 +146,19 @@ def process(self, instance):
             "deadlineChunkSize": deadlineChunkSize,
             "deadlinePriority": deadlinePriority
         })
-
+        self.log.debug("families: {}".format(families))
+        if "prerender" in families:
+            _families = list()
+            for fm in families:
+                if fm in _families:
+                    continue
+                if "render" in fm:
+                    if "prerender" in fm:
+                        continue
```

Comment:
This is a bit convoluted. We have the node marked with prerender or render family and it should never have both.